src/rdgnu/public/vercel.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="283" height="64" viewBox="0 0 283 64" fill="none" 
```

Comment:
I think we can probably get rid of this and the references to it

Reply:
+1, let's clean up the NextJS default stuff before we merge.